helpers.go

```diff
@@ -349,12 +350,26 @@ func (gb *GoBrew) downloadAndExtract(version string) {
 func (gb *GoBrew) changeSymblinkGoBin(version string) {
 	goBinDst := filepath.Join(gb.versionsDir, version, "/go/bin")
 	_ = os.RemoveAll(gb.currentBinDir)
+	if runtime.GOOS == "windows" {
```

Comment:
It is also necessary to remove the checks on the operating system and add a function like `symlink`, which we will implement differently depending on the type of operating system in our file.

Again, because I don't have windows at hand, I need a little more time.

And the question is, why in one case do you call ps, and here you call cmd? Why such a spread?

Reply:
To my knowledge, there's no easy way to eval a symlink with `cmd`, and I use `cmd` because it doesn't require privilege to create symlinks.

I'll remove the GOOs check, I've never coded cross-platform code before.

Reply:
![image](https://github.com/kevincobain2000/gobrew/assets/20581504/6e1799da-333c-4635-bb68-9de409ca71d3)